card/lib/cardio.rb

```diff
@@ -60,6 +60,7 @@ def default_configs
         file_storage:           :local,
         file_buckets:           {},
         file_default_bucket:    nil,
+        rich_text_editor:       :prosemirror,
```

Comment:
I think we should return to tinymce as default editor for now. The current version has improved a lot compared to what we were using before we switched to prosemirror. Prosemirror is probably more powerful in the long term (eg. regarding simultaneous editing) but without any plugins the ui is very limited. There are almost no plugins available yet and we don't have the resources to help to change that. PR with tinymce update is coming.